### PR TITLE
Fixed all references in documentation part I of PageBase and removed Base.

### PR DESCRIPTION
**Pull request by StevenOwenNell**

### CODE REVIEW

**Comment by hikalkan** [comment]
Thanks a lot.

File: docs/en/Tutorials/AspNetCore-Mvc/Part-I.md [hunk 1]
```diff
@@ -337,13 +337,13 @@ Open the `Index.cshtml` and change the content as shown below:
...
-@inherits Acme.BookStore.Web.Pages.BookStorePageBase
+@inherits Acme.BookStore.Web.Pages.BookStorePage
 @model IndexModel
 
 <h2>Books</h2>
 ````
 
-* This code changes the default inheritance of the Razor View Page Model so it **inherits** from the `BookStorePageBase` class (instead of `PageModel`).  The `BookStorePageBase` class which comes with the startup template and provides some shared properties/methods used by all pages.
+* This code changes the default inheritance of the Razor View Page Model so it **inherits** from the `BookStorePage` class (instead of `PageModel`).  The `BookStorePage` class which comes with the startup template and provides some shared properties/methods used by all pages.
 * Ensure that the `IndexModel` (*Index.cshtml.cs)* has the `Acme.BookStore.Pages.Books` namespace, or update it in the `Index.cshtml`.
```
Comment (cesardv) [inline docs/en/Tutorials/AspNetCore-Mvc/Part-I.md:347]: This commit was a while ago, but shouldn't this say "has the Acme.BookStore.*Web*.Pages.Books namespace..."?

Comment (hikalkan) [reply docs/en/Tutorials/AspNetCore-Mvc/Part-I.md:347]: @maliming can you check and fix the documentation if necessary.
Thanks @cesardv

Comment (maliming) [reply docs/en/Tutorials/AspNetCore-Mvc/Part-I.md:347]: hi @cesardv 

We no longer use `@inherits` in the page.
https://github.com/abpframework/abp/pull/4247

**Pull request merged**